config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
That looks the wrong way round to me, because you're still coupling generic KVM setup config to a specific scenario, and then coupling other scenarios to that.  So for example if you later wanted to remove `create_kvm_image` then it's a mess because you would have to relocate that config to somewhere else.  If you wanted to do it using YAML anchors and refs then something like this would be cleaner:

```yaml
features:
  glance:
    flags: ""
    kvm_image: &kvm_image
      image_name: jeos1
      image_source: http://clouddata.cloud.suse.de/images/jeos-64.qcow2
      properties: "--property hypervisor_type=kvm"
    create_kvm_image:
    <<: *kvm_image
    scenario_nr_2:
    <<: *kvm_image
```